src/extension_mainfn.py

```
@@ -1,6 +1,8 @@
 def main():
     return True
+
+
 # Appended extension code from GitHub:
 def main():
```

Ruff (0.11.9)

Redefinition of unused `main` from line 1

(F811)